tests/js/realm-tests.js

```diff
@@ -38,21 +38,44 @@ const Utils = require('./test-utils');
 
 let pathSeparator = '/';
 const isNodeProcess = typeof process === 'object' && process + '' === '[object process]';
+const isElectronProcess = typeof process === 'object' && process.versions && process.versions.electron;
+
 if (isNodeProcess && process.platform === 'win32') {
     pathSeparator = '\\';
 }
 
 const fs = isNodeProcess ? nodeRequire('fs-extra') : require('react-native-fs');
 
 module.exports = {
-    testRealmConstructor: function() {
+    testRealmConstructorTest: function() {
```

Comment:
This was added so filtering is better. Otherwise you can not filter and run this test only. We should be using unique method names for each test so you can debug that single test easily by changing the filter with that name. Jasmine filters will run every test that starts with `testRealmConstructor`. Hence the name change